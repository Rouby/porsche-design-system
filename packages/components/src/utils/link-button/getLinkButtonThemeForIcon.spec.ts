import { getLinkButtonThemeForIcon } from './getLinkButtonThemeForIcon';

describe('getLinkButtonThemeForIcon()', () => {
  it.each<Parameters<typeof getLinkButtonThemeForIcon>>([
    ['primary', 'light'],
    ['primary', 'dark'],
    ['secondary', 'light'],
    ['secondary', 'dark'],
    ['tertiary', 'light'],
    ['tertiary', 'dark'],
  ])('should return correct theme for variant: %s and theme: %s', (...args) => {
    expect(getLinkButtonThemeForIcon(...args)).toMatchSnapshot();
  });
});
