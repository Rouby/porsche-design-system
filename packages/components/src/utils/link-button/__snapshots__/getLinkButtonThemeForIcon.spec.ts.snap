// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getLinkButtonThemeForIcon() should return correct theme for variant: primary and theme: dark 1`] = `"light"`;

exports[`getLinkButtonThemeForIcon() should return correct theme for variant: primary and theme: light 1`] = `"dark"`;

exports[`getLinkButtonThemeForIcon() should return correct theme for variant: secondary and theme: dark 1`] = `"dark"`;

exports[`getLinkButtonThemeForIcon() should return correct theme for variant: secondary and theme: light 1`] = `"light"`;

exports[`getLinkButtonThemeForIcon() should return correct theme for variant: tertiary and theme: dark 1`] = `"dark"`;

exports[`getLinkButtonThemeForIcon() should return correct theme for variant: tertiary and theme: light 1`] = `"light"`;
